std_msgs/UInt32 origin_image_width
std_msgs/UInt32 origin_image_height
iana_text_recognition/TextWithROI[] texts_in_image
